src/book/mod.rs

```diff
@@ -270,8 +271,13 @@ impl MDBook {
                     _ => continue,
                 };
 
+                if !chapter.is_empty() && ch.name != chapter {
+                    info!("Skipping chapter '{}'...", ch.name);
```

Comment:
I'm think it might be better to make this `debug!`.  In a larger book, it will flood the console with a bunch of skipping messages that obscure what it is doing.

Reply:
Done.